docs/concepts.md

```diff
@@ -133,23 +155,7 @@ There are lot of interesting things happening in this example, which bring out t
  strategy, where we aggressively compact the latest partitions compared to older partitions, we could ensure the RO Table sees data
  published within X minutes in a consistent fashion.
 
-The intention of merge on read storage, is to enable near real-time processing directly on top of Hadoop, as opposed to copying
+The intention of merge on read storage, is to enable near real-time processing directly on top of DFS, as opposed to copying
```

Comment:
The primary intention of merge on read storage is to enable near real-time processing directly on top of DFS....
There are few secondary side benefits to this storage such as reduced write amplification by avoiding synchronous merge of data, i.e, the amount of data written per 1 bytes of data in a batch